multiregion/scripts/run-consumer.sh

```diff
@@ -16,3 +16,24 @@ docker-compose exec broker-east-3 kafka-consumer-perf-test --topic multi-region-
     --broker-list broker-west-1:19091,broker-east-3:19093 \
     --timeout 20000 \
     --consumer.config /etc/kafka/demo/consumer-east.config
+
+docker-compose exec broker-east-3 kafka-consumer-perf-test --topic multi-region-async-op-under-min-isr \
```

Comment:
@thomaskwscott is it intentional to add consume statements for all the new topics?  Can we remove them?  The reason being:

1. The existing consume from `multi-region-async` is enough to show performance
2. It changes the docs output

Reply:
absolutely we can remove them. They were only there for completeness